examples/parametric/plot_equivalent_width.py

```diff
@@ -151,9 +97,9 @@ def equivalent_width(grids, uv_index, index_window, blue_window, red_window):
         feature, blue, red = index_window[i], blue_window[i], red_window[i]
 
         for k in range(0, len(Z)):
-            EqW = get_ew(index, feature, blue, red, Z[k], stellar_mass, grid, EqW, 0)
+            EqW = measure_equivalent_width(index, feature, blue, red, Z[k], stellar_mass, grid, EqW, 0)

```

Comment:
Avoid capitalisation of variables. These should adhere to snake_case (I.e. all lowercase and as descriptive as possible with underscores).

Reply:
Noted, I'll fix that right away